aptos-move/e2e-move-tests/src/aggregator_v2.rs

```diff
@@ -0,0 +1,193 @@
+// Copyright © Aptos Foundation
```

Comment:
I think we can reduce duplication between v1 and v2, can be done later in other PR though

Reply:
@vusirikala let's reduce duplication here, e.g. `initialize` function can be shared with the initial aggregator file, otherwise it is hard to sync changes in the future, see my comment for `initialize`.

Reply:
@georgemitenkov aggregator.rs and aggregator_v2.rs may look the same. But aggregator.vs is initializing aggregator v1 and storing in AggregatorStore. aggregator_v2.rs is initializing aggregator from aggregator v2 module and storing in AggregatorStore. They work differently.